src/ActionsApple.cpp

```diff
@@ -0,0 +1,120 @@
+#include "Actions.hpp"
+#include "MemAllocHeap.hpp"
+#include "StatCache.hpp"
+#include "BinaryData.hpp"
+
+#if defined(TUNDRA_APPLE)
+
+#include <copyfile.h>
+#include <sys/time.h>
+#include <sys/stat.h>
+
+ExecResult CopyFiles(const FrozenFileAndHash* src_files, const FrozenFileAndHash* target_files, size_t files_count, StatCache* stat_cache, MemAllocHeap* heap)
+{
+    ExecResult result;
+    memset(&result, 0, sizeof(result));
+    char tmpBuffer[1024];
```

Comment:
(bit of a nitpick) I wonder if we should file an issue to remove the use of fixed size buffers?